src/redux-middleware/scrollCursorIntoView.ts

```diff
@@ -19,6 +24,26 @@ editingValueStore.subscribe(
   }, 400),
 )
 
+/**
+ * Scroll the cursor into view if the sync was the result of navigation to a thought.
+ */
+syncStatusStore.subscribe(state => {
+  if (!state.isPulling && navigated) {
```

Comment:
We only want this triggered when `isPulling` *changes* to false, not every time it is false. The `syncStatusStore` can update for other reasons.

You can either modify the ministore to pass the old state to `subscribe`, or you can just add an `isPullingPrev` variable here to hold onto the previous value.